templates/event_checkin.html.twig

```diff
@@ -13,7 +13,7 @@
             <div data-user-id="{{ rsvp.id }}"
                  data-user-name="{{ rsvp.name }}"
                  class="rsvp {{ rsvp.id in checkins or rsvp.host ? 'checked_in':'not_checked_in' }}"
-                 style="background-image: url('{{ rsvp.photo.photo_link|default('/img/avatar.png') }}');"
+                 style="background-image: url('{{ rsvp.photo.photo_link|default(asset('assets/avatar.png') }}');"
```

Comment:
```suggestion
                 style="background-image: url('{{ rsvp.photo.photo_link|default(asset('assets/avatar.png')) }}');"
```

Reply:
Nice catch! I've added Twig linting to `make test`, which is also ran by CircleCI, to prevent similar errors in the future.